components/ws-manager-mk2/controllers/workspace_controller_test.go

```diff
@@ -209,18 +220,19 @@ var _ = Describe("WorkspaceController", func() {
 	})
 })
 
-func updateObjWithRetries[O client.Object](obj O, updateStatus bool, update func(obj O)) {
+func updateObjWithRetries[O client.Object](c client.Client, obj O, updateStatus bool, update func(obj O)) {
+	GinkgoHelper()
```

Comment:
`GinkgoHelper()` results in more usable stacktraces if a test fails, the resulting stacktrace points to the line where the helper method is called from